devlib/utils/rendering.py

```diff
@@ -83,8 +85,12 @@ def write_frames(self, outfile, columns=None):
             header = self.header
             frames = self.frames
         else:
-            header = [c for c in self.header if c in columns]
-            indexes = [self.header.index(c) for c in header]
+            indexes = []
+            for c in columns:
+                if c not in self.header:
+                    msg = 'Invalid column "{}"; must be in {}'
+                    raise ValueError(msg.format(c, self.header))
+                indexes.append(self.header.index(c))
```

Comment:
I think 
```
header = columns
```
 is missing

Reply:
Ah yes. Thanks for catching that! Fixed.